### PR TITLE
download: use net/http instead of `wget` by default.

### PR DESCRIPTION
**Pull request by cubicdaiya**
refs #1

### CODE REVIEW

File: download.go [hunk 1]
```diff
@@ -16,7 +19,32 @@ func extractArchive(path string) error {
 	return command.Run([]string{"tar", "zxvf", path})
 }
 
-func download(url string, logName string) error {
+func downloadBuiltin(url string, logName string) error {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Accept-Encoding", "gzip")
```
Comment (catatsuy) [inline download.go]: Why is this necessary? This header is added by `net/http.Transport`.

Comment (cubicdaiya) [inline download.go]: It's my mistake. Thanks. BTW, this PR is not active.

**Comment by cubicdaiya** [comment]
Continue #25
